src/sink/elastic.rs

```diff
@@ -74,14 +74,14 @@ impl offramp::Impl for Elastic {
                 .build()?;
 
             let queue = AsyncSink::new(config.concurrency);
-            let (tx, rx) = bounded(crate::QSIZE);
+            let (tx, _rx) = bounded(1); // dummy value
 
             Ok(SinkManager::new_box(Self {
+                sink_url: TremorURL::from_offramp_id("elastic")?, // just a dummy value, gonna be overwritten on init
                 postprocessors: vec![],
                 client,
                 queue,
-                rx,
-                tx,
+                insight_tx: tx,
```

Comment:
Oops, i moved elastic from the old `drain_insights` pattern to emit insights immediately.